activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeBPMNMessageIT.java

```diff
@@ -122,20 +93,23 @@ public void shouldThrowIntermediateMessageEvent() throws Exception{
                                                                             .withBusinessKey("businessKey")
                                                                             .withProcessDefinitionKey(PROCESS_INTERMEDIATE_THROW_MESSAGE_EVENT)
                                                                             .build());
-        assertThat(receivedEvents)
+        assertThat(MessageTestConfiguration.messageEvents)
                 .isNotEmpty()
-                .extracting("type", 
-                            "processDefinitionId",
-                            "processInstanceId",
-                            "activityType",
-                            "messageName",
-                            "messageCorrelationKey",
-                            "messageBusinessKey",
-                            "messageData")
-                .contains(Tuple.tuple(ActivitiEventType.ACTIVITY_MESSAGE_SENT,
+                .extracting(BPMNMessageEvent::getEventType,
+                            BPMNMessageEvent::getProcessDefinitionId,
+                            BPMNMessageEvent::getProcessInstanceId,
+                            event -> event.getEntity().getProcessDefinitionId(),
+                            event -> event.getEntity().getProcessInstanceId(),
+                            event -> ((BPMNMessage)(event.getEntity())).getMessagePayload().getName(),
+                            event -> ((BPMNMessage)(event.getEntity())).getMessagePayload().getCorrelationKey(),
+                            event -> ((BPMNMessage)(event.getEntity())).getMessagePayload().getBusinessKey(),
+                            event -> ((BPMNMessage)(event.getEntity())).getMessagePayload().getVariables()  
+                )
+                .contains(Tuple.tuple(BPMNMessageEvent.MessageEvents.MESSAGE_SENT,
+                                      process.getProcessDefinitionId(),
+                                      process.getId(),
                                       process.getProcessDefinitionId(),
                                       process.getId(),
-                                      "throwEvent",
                                       "Test Message",
```

Comment:
We seem to have lost activity type and other attributes from original engine event, i. e. process businessKey. It would be useful info for downstream consumers I think.

Reply:
For events we have a payload where we store this info:
https://github.com/Activiti/Activiti/pull/2898/files#diff-01257cc01747b6a40cd9228263a55296R37
I think in cloud events (later) we store such kind of info for events...

But I agree, that we may need activity type for this event... Should I add this to API?

Reply:
I think this requires global attribute `activityType` which is out of the scope of this PR